macos/set-defaults.sh

```diff
@@ -169,6 +169,19 @@ defaults write com.apple.finder FXInfoPanesExpanded -dict \
 	OpenWith -bool true \
 	Privileges -bool true
 
+###############################################################################
+# Screenshots                                                                 #
+###############################################################################
+
+# Set default screenshot location
```

Comment:
Could you remove this setting. I actually like them being created on the desktop.